package.json

```diff
@@ -7,13 +7,14 @@
   "author": "ArtemZayarny <artyom.zayarny@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "dev": "nodemon ./src/index.ts",
+    "dev": "nodemon ./src/rabbitmq/rabbitmq.consumer.ts && ./src/index.ts",
```

Comment:
This approach might be not the best as `nodemon` will monitor only the first process.. just something to think about. You could either have 2 separate commands for starting 2 services, or use some other npm package or maybe even consider using Docker and docker-compose.